handler/core/README.md

```diff
@@ -190,3 +190,19 @@ product-cache {
 Please note that cacheKey can be parametrized with request data like params, headers etc. Read 
 [Knot.x HTTP Server Common Placeholders](https://github.com/Knotx/knotx-server-http/tree/master/common/placeholders)
 documentation for more details. 
+
+## Debug mode
+> Notice!<br>
```

Comment:
Can you please add some assets that presents Debug mode in action?

Reply:
added